modules/main/start/keyboard.py

```diff
@@ -1,45 +1,28 @@
-from aiogram.types import InlineKeyboardButton, InlineKeyboardMarkup, Message, CallbackQuery
+from aiogram.types import InlineKeyboardButton, InlineKeyboardMarkup, Message, CallbackQuery, WebAppInfo
 from aiogram.utils.keyboard import InlineKeyboardBuilder
 
-from utils import Translator
 
+def language(language_code: str) -> str:
```

Comment:
You can remove this function from keyboard.py. `locale` in `keyboard()` is result of `language()`.